bundles/org.openhab.binding.nanoleaf/src/main/java/org/openhab/binding/nanoleaf/internal/handler/NanoleafPanelHandler.java

```diff
@@ -286,98 +276,33 @@ public void updatePanelGesture(int gesture) {
         }
     }
 
-    public String getPanelID() {
+    public Integer getPanelID() {
```

Comment:
Would primitive type `int` be sufficient here?

Reply:
It would. But it is an Integer in the configuration, and it is used to look up in some other maps as an Integer, so I was thinking keeping it as Integer would save us from some unboxing/boxing?

Reply:
When propving to myself that the config could only be Integer, I found NanoleafPanelConfig.  For me it looks like it is not used. Could it be that this is some historic artifact and was used before openHAB used Maps for config?

Reply:
It looks like you are right, so it could be removed. I don't know the reason for it not being used, it's still common to use a DTO for configuration.

Reply:
Since it does not cause harm, let's merge. It can be refactored later.